### ISSUE TITLE
Retrain the Deepfilternet3 in released config.ini

### ISSUE THREAD

**Issue by Leaou-Bin** [Rikorose/DeepFilterNet#269]
Thanks for your excellent work!
I'm trying to retrain the DeepfilterNet3 within the configuraion in models/DeepFilterNet2/config.ini, and test the trained model from df/enhance.py, the output wavs is worse than the released model in models/DeepFilterNet2/, the difference in high frequency component is obvious.
The train data,and train log.
(1) Train clean speech and noise: 2021DNS_48k
(2) It early stop at epoch 91,Found checkpoint model_77.ckpt.best with epoch 77,the training loss finally converges to about 1.46,test loss:  1.33637 
Here is some samples we tested,the initial wav is 16k,using resample(audio, df_sr, 48000) first, then resample and save in 16k, following is a comparison of the spec.
Enhanced spec from model in models/DeepFilterNet2/model_96.ckpt.best
![image](https://user-images.githubusercontent.com/55938719/222062821-b8946f3e-7fa7-4b6a-88a1-7199efa66e6c.png)
Retrained DeepFilterNet3
![image](https://user-images.githubusercontent.com/55938719/222063233-dedfe053-4132-4278-880d-4d8996a4794f.png)

Could you give some advice on whether the model converges and more tips on training ?

**Comment by Rikorose** [comment]
I guess you have the most chances working on your data. First explicitly oversample VCTK and PTDB or other high quality 48kHz datasets. Further, the raw DNS datasets includes data that had been resampled/compressed at some point. For the DeepFilterNet3 model, I also filtered the DNS dataset with DNSMOS to further filter out bad quality samples.

**Comment by Leaou-Bin** [comment]
Besides,we also trained the DeepfilterNet3 in 16k, using WSJ0 + DNS noise,and test result is not so good either,and there is a OOM error while training DeepfilterNet2 in linux + cu117+torch1.13,whether DeepfilterNet2 may work better on 16k sample rate.
I read the issue that you haven't carryied it in 16k,but can you give some personal insights?

**Comment by Rikorose** [comment]
I only tried 24kHz and this worked well. You may need to adjust f_df, nb_erb etc.  OOM also depend on hardware, batch size, tbpt length etc.

**Comment by JBloodless** [comment]
> I only tried 24kHz and this worked well. You may need to adjust f_df, nb_erb etc. OOM also depend on hardware, batch size, tbpt length etc.

Can you please share your parameters for 24khz training? I've tried my own based on my 48k and 16k setups, and for some reason result is way worse. Maybe I did something wrong elsewhere, but I'd be glad to compare with yours. Mine are
```
sr = 24000
fft_size = 720
hop_size = 360
nb_erb = 24
nb_df = 72
```

**Comment by Rikorose** [comment]
I trained using a low latency configuration for hearing aids:

```
[df]
sr = 24000
fft_size = 96
hop_size = 24
nb_erb = 16
nb_df = 16
norm_tau = 1
lsnr_max = 35
lsnr_min = -15
min_nb_erb_freqs = 2
pad_mode = output
df_order = 5
df_lookahead = 2
